ci-scripts/validate-lint.sh

```diff
@@ -53,7 +53,7 @@ fi
 validateStylesLint
 
 echo "Validating code linting"
-ng lint
+node --max_old_space_size=3584 ./node_modules/@angular/cli/bin/ng lint
```

Comment:
@sam-garland 
Can you confirm if we still really need this change?

I reacall you had some problems on CI November 19th. But since then you made a lot of changes around the usage of ui5 package. And I'm just wondering if it's still needed.

Reply:
We do still run out of RAM when the V8 default behaviour is used (in which case an OOM error occurs when the heap goes over 2GB on a travis node with 4GB of RAM).

https://app.travis-ci.com/github/SAP/spartacus/jobs/553157464

The other way of specifying this is to use the NODE_OPTIONS environment variable:
```bash
NODE_OPTIONS=--max_old_space_size=3584 ng lint
```